assets/scss/generic-overrides.scss

```diff
@@ -194,6 +194,13 @@ a:focus .nhsuk-icon {
 @media (min-width: 40.0625em) {
 	.nhsuk-header__container {
         justify-content: normal;
+		.nhsuk-header__logo {
```

Comment:
Why do we need it? The standard NHSUK Header is fine as it is ... See below 
![image](https://github.com/user-attachments/assets/f25ea76f-638f-423e-91da-55ff8a0c6c28)

Reply:
This seems to break if we have larger names. 
<img width="1665" alt="Screenshot 2024-12-18 at 12 02 34" src="https://github.com/user-attachments/assets/8f62d662-0dc2-4f7e-9c50-1ab544ead2e0" />